src/Circle.js

```diff
@@ -0,0 +1,95 @@
+import React, { useRef, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const DIRECTIONS = {
+    CLOCKWISE: 'clockwise',
+    ANTI_CLOCKWISE: 'antiClockwise',
+    BOTH_SIDES: 'bothSides',
+};
+
+const getSvgClassName = (strokePercentage, direction, className) => classNames(
+    'next-button_svg',
```

Comment:
Why `next-button_*`? 🤔

This has to be `react-circle_*`.

Reply:
🤦 you're absolutely right